### DISCUSSION TITLE
2.3.70 Errors detected during installation - Sensor Node Install

### DISCUSSION THREAD

**Discussion by Rbartram** [Security-Onion-Solutions/securityonion#5386] [Unsupported Versions] [answered]
Hello, 

When installing a Sensor node from ISO, the installation ends with an error.

What I think are relevant entries from sosetup.log:
   ----------
   8098 [INFO    ] Running state [wazuh-agent] at time 13:57:42.962687
   8099 [INFO    ] Executing state service.running for [wazuh-agent]
   8100 [INFO    ] Executing command systemctl in directory '/root'
   8101 [ERROR   ] Failed to get information on unit 'wazuh-agent'
   8102 [INFO    ] Completed state [wazuh-agent] at time 13:57:42.980085 (duration_in_ms=17.4)
   ...
  11739 Summary for local
  11740 --------------
  11741 Succeeded: 212 (changed=55)
  11742 Failed:      1
  11743 --------------
  11744 Total states run:     213
  11745 Total run time:   220.090 s
  11746 Errors detected during setup; skipping post-setup steps to allow for analysis of failures.

   
   
What I think is relevant output from salt-call state.highstate:
   ----------
          ID: wazuhagentservice
    Function: service.running
        Name: wazuh-agent
      Result: False
     Comment: Failed to get information on unit 'wazuh-agent'
     Started: 14:41:10.497649
    Duration: 32.486 ms
     Changes:

   

Output from systemctl status wazuh-agent.service:
   ----------
    Failed to get properties: Access denied

I welcome advise.

Thank you, ...Rob

**Comment by Rbartram** [comment]
It seems a reboot cleared up the wazuhagentservice failure reported by salt-call state.highstate.

so-status is showing that all container statuses are OK and the sensor node is now showing as OK in the grid.

Are there any post-setup steps that I need to revisit?

**Reply by 3isenHeiM** [reply]
I got the same status with a Manager running the version 2.3.180 and indeed a reboot solved the issue.

**Comment by dougburks** [comment, marked as answer]
It seems like you were using an older 2.3.70 ISO image release before Tuesday 8/31/2021. From https://blog.securityonion.net/2021/08/security-onion-2370-wazuh-hotfix-now.html:

> After we released Security Onion 2.3.70, there was a change on the upstream Wazuh repo that caused our installer to attempt to update Wazuh which resulted in an error. We've added some logic to prevent this from happening.

**Reply by Rbartram** [reply]
Ahh, that makes sense.  It looks like I just need to run soup to install updates.  Is that correct?

**Reply by TOoSmOotH** [reply]
You don't need to re-install just soup the manager and you should be fine.

**Reply by Rbartram** [reply]
Thank you!
